2 - RTB/Documentazione esterna/Analisi dei Requisiti/Analisi dei Requisiti.typ

```diff
@@ -344,12 +327,12 @@ $bold("Postcondizioni: ")$
 - all'utente viene notificato l'errore relativo ad un'immissione errata dei dati per la modifica dell'ambiente.
 
 $bold("Scenario: ")$
-- l'utente ha immesso dei dati errati per la modifica dell'ambiente.
+- l'utente ha immesso delle dimensioni troppo piccole nella modifica dell'ambiente, rischiando di generare conflitti.
```

Comment:
(continuo dalla conversation di prima) inserendo nuove dimensioni e creando un nuovo ambiente non dovrebbe anche eliminare tutti gli scaffali inseriti e far ripartire tutto da capo?

Reply:
Rispondo qui visto che è una continuazione del commento precedente.

Probabilmente sono io che ho capito male, perché l'idea che mi ero fatto era che:
- utente crea magazzino;
- utente modifica magazzino;
- viene apportata la modifica delle dimensioni **senza fare un restart**.
Il caso di inserimento di dimensioni troppo piccole, quindi, si ha perché se l'utente inserisce delle dimensioni che andrebbero a tagliare fuori degli scaffali durante la modifica è come se si entrasse nel caso di eliminazione scaffale non vuoto, che era considerato una situazione di errore (UC-3.4.1).
Tutto ciò, però, deriva dal fatto che ho considerato la modifica come un'azione che **non** fa ripartire tutto da capo eliminando l'ambiente e ricreandolo, ma va solamente ha cambiare ciò che già esiste.

Reply:
ok, era per capire, dato che quando ero analista io c'era un'idea differente a riguardo. Mi fido del fatto che questa cosa nasca comunque da una vostra riflessione in merito e per me non c'è problema.